src/main/webapp/src/app/location/location.component.html

```diff
@@ -1,9 +1,19 @@
 <div id="location-container">
     <h1>Welcome to Hidden Gems!</h1>
     <h4>To use Hidden Gems, please allow location services. If not possible select a default location [Sydney, Adelaide].</h4>
-    <button id="geolocationButton" (click)="enableNavigatorLocation()" type="button"><i></i>Find My Location</button>
-    <button id="sydneyButton" (click)="getSydneyLocation()" type="button"><i></i>Sydney</button>
-    <button id="adelaideButton" (click)="getAdelaideLocation()" type="button"><i></i>Adelaide</button>
+    
+    <mat-card>
+      <button mat-raised-button color="primary" id="geolocationButton" (click)="enableNavigatorLocation()" type="button"><i></i>Find My Location</button>
+    </mat-card>
+
+    <mat-card>
+      <button mat-button [matMenuTriggerFor]="menu">Select your city ⬇️</button>
+      <mat-menu #menu="matMenu">
```

Comment:
Maybe use a mat-select component so that you can have the pretty arrow of Angular material? 
Source: https://material.angular.io/components/select/overview